charts/bulk-scan-orchestrator/values.yaml

```diff
@@ -31,6 +31,8 @@ java:
         - idam-users-div-password
         - idam-users-probate-username
         - idam-users-probate-password
+        - idam-users-publiclaw-username
+        - idam-users-publiclaw-password
```

Comment:
those only exist in aat. will service be ok to launch without secrets being present in other environments?

Reply:
prod pod will fail if secrets are not there so we should add them I think in all environments.
But I agree we should just enable them in AAT https://github.com/hmcts/bulk-scan-orchestrator/blob/master/infrastructure/aat.tfvars#L4

Reply:
ok this is now deprecated so secrets has to exist in prod.

Reply:
I added supported jurisdictions config to be able to disable publiclaw in prod. But, we still need to add idam credentials to prod key vault

Reply:
Updated  prod config to exclude `publilaw` https://github.com/hmcts/cnp-flux-config/pull/2169/files#diff-a81b4fc624fb7995c6f17e36a59a0540R28

Reply:
The idam secrets are now added to prod vault

Reply:
i presume perftest can be done together with the ticket responsible deploying orchestrator to perftest